Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII_test/TestCommands/indicators_example.json

```diff
@@ -0,0 +1,167 @@
+        "stixindicatordescription": "This URL:[http://example.org/] was identified by phishtank.com as part of a phishing email. This URL appears to still be online as of 2021-10-19T10:07:34+00:00. More detailed infomation can be found at http://www.phishtank.com/phish_detail.php?phish_id=7324767",
+        "confidence": "High",
+        "fields": {},
+        "relationships": [
+            {
+                "name": "related-to",
+                "reverseName": "related-to",
+                "type": "IndicatorToIndicator",
+                "entityA": "http://example.org/",
+                "entityAFamily": "Indicator",
+                "entityAType": "URL",
+                "entityB": "URL embedded in Email",
+                "entityBFamily": "Indicator",
+                "entityBType": "Attack Pattern",
+                "fields": {}
+            }
+        ],
+        "rawJSON": {
+            "indicator": "http://example.org/",
+            "type": "URL",
+            "indicator_ref": "opensource:Observable-cfe14953-f986-450b-93f1-0041a597b8d2",
+            "stix_title": "URL: http://example.org/...",
+            "stix_description": "URL: http://example.org/| isOnline:yes| dateVerified:2021-10-19T10:07:34+00:00",
+            "TLP": "WHITE",
+            "share_level": "white",
+            "stix_indicator_name": "phishTank.com id:7324767 with malicious URL:https://example...",
+            "stix_indicator_description": "This URL:[http://example.org/] was identified by phishtank.com as part of a phishing email. This URL appears to still be online as of 2021-10-19T10:07:34+00:00. More detailed infomation can be found at http://www.phishtank.com/phish_detail.php?phish_id=7324767",
+            "confidence": "High",
+            "ttp_ref": [
+                "opensource:ttp-c819f3ef-fbc3-4077-8d56-bf619c8d9b29"
+            ],
+            "relationships": [
+                {
+                    "stix_ttp_title": "Email Emmbedded URL",
+                    "ttp_description": "Target Users via Email by adding a malicious URL",
+                    "type": "Attack Pattern",
+                    "indicator": "URL embedded in Email",
+                    "value": "URL embedded in Email"
+                }
+            ],
+            "value": "http://example.org/"
+        }
+    },
+    {
+        "value": "URL embedded in Email",
+        "type": "Attack Pattern",
+        "title": null,
+        "description": null,
+        "shortdescription": null,
+        "stixindicatordescription": "Target Users via Email by adding a malicious URL",
+        "stixttptitle": "Email Emmbedded URL",
+        "score": 2,
+        "fields": {},
+        "rawJSON": {
+            "stix_ttp_title": "Email Emmbedded URL",
+            "ttp_description": "Target Users via Email by adding a malicious URL",
+            "type": "Attack Pattern",
+            "indicator": "URL embedded in Email",
+            "value": "URL embedded in Email"
+        }
+    }
+]
```

Comment:
Same here - check for secrets

Reply:
<img width="1543" alt="Screen Shot 2021-11-07 at 12 42 51" src="https://user-images.githubusercontent.com/78307768/140641686-764ac087-a190-4b1d-b839-dc4234105a48.png">